cmp/grid/filter/GridFilterFieldSpec.ts

```diff
@@ -42,7 +41,7 @@ export interface GridFilterFieldSpecConfig extends BaseFilterFieldSpecConfig {
 export class GridFilterFieldSpec extends BaseFilterFieldSpec {
     filterModel: GridFilterModel;
     renderer: ColumnRenderer;
-    inputProps: PlainObject;
+    inputProps: HoistInputProps;
```

Comment:
See #3510 - @saba-mo and I just ran into that issue last week, would be great if we could address as part of this PR, taking this one step further to support a broader union type here and on the `GridFilterFieldSpecConfig` interface

Reply:
will hold off on this change then, and take the other simpler changes tin this PR